src/it/resources/com/google/checkstyle/test/chapter5naming/rule523methodnames/InputMethodName.java

```diff
@@ -8,44 +8,44 @@
  */
 public class InputMethodName {
     void foo() {}
-    void Foo() {} //warn
-    void fOo() {} //warn
+    void Foo() {} // violation 'Method name 'Foo' must match pattern .*'
+    void fOo() {} // violation 'Method name 'fOo' must match pattern .*'
     void f0o() {}
-    void f$o() {} //warn
-    void f_oo() {} //warn
-    void f() {} //warn
-    void fO() {} //warn
+    void f$o() {} // violation 'Method name '.*' must match pattern .*'
+    void f_oo() {} // violation 'Method name 'f_oo' must match pattern .*'
+    void f() {} // violation 'Method name 'f' must match pattern .*'
+    void fO() {} // violation 'Method name 'fO' must match pattern .*'
 
     class InnerFoo {
         void foo() {}
-        void Foo() {} //warn
-        void fOo() {} //warn
+        void Foo() {} // violation 'Method name 'Foo' must match pattern .*'
+        void fOo() {} // violation 'Method name 'fOo' must match pattern .*'
         void f0o() {}
-        void f$o() {} //warn
-        void f_oo() {} //warn
-        void f() {} //warn
-        void fO() {} //warn
+        void f$o() {} // violation 'Method name '.*' must match pattern .*'
+        void f_oo() {} // violation 'Method name 'f_oo' must match pattern .*'
+        void f() {} // violation 'Method name 'f' must match pattern .*'
+        void fO() {} // violation 'Method name 'fO' must match pattern .*'
     }
 
     InnerFoo anon = new InnerFoo() {
         void foo() {}
-        void Foo() {} //warn
-        void fOo() {} //warn
+        void Foo() {} // violation 'Method name 'Foo' must match pattern .*'
+        void fOo() {} // violation 'Method name 'fOo' must match pattern .*'
         void f0o() {}
-        void f$o() {} //warn
-        void f_oo() {} //warn
-        void f() {} //warn
-        void fO() {} //warn
+        void f$o() {} // violation 'Method name '.*' must match pattern .*'
```

Comment:
No need in starting/training `.*` it is implied, remove all

Reply:
Removed the ending `.*`, I didn't not removed the starting `.*` because without it the test case was failing:

https://github.com/checkstyle/checkstyle/blob/daeba08b4cdf6791745625ec4a6dcbdc7ff9dde2/src/it/resources/com/google/checkstyle/test/chapter5naming/rule523methodnames/InputMethodName.java#L35